castle/src/main/java/io/castle/android/CastleConfiguration.java

```diff
@@ -86,6 +99,25 @@ public boolean screenTrackingEnabled() {
         return screenTrackingEnabled;
     }
 
+    /**
+     * Get use cloudflare app enabled
+     * @return Use Cloudflare app enabled
+     */
+    public boolean useCloudflareApp() { return useCloudflareApp; }
+
+    /**
+     * Get API domain
+     * @return API domain
+     */
+    public String apiDomain() { return apiDomain; }
+
+    public String baseUrl() {
+        if (useCloudflareApp) {
+            return String.format("https://%s/%s", apiDomain, CLOUDFLARE_API_PATH);
```

Comment:
Instead of CLOUDFLARE_API_PATH, it could be also another path provided by the user (v1/c/mobile/ is the default one)